repose-aggregator/artifacts/src/config/filters/log4j2.xml

```diff
@@ -26,9 +26,23 @@
         <Logger name="com.sun.jersey" level="off"/>
         <Logger name="net.sf.ehcache" level="error"/>
         <Logger name="org.apache" level="warn"/>
+        <Logger name="org.apache.http.wire" level="trace">
+            <ContextMapFilter onMatch="ACCEPT" onMismatch="DENY">
+                <KeyValuePair key="x-trace-request" value="true"/>
+            </ContextMapFilter>
+        </Logger>
         <Logger name="org.eclipse.jetty" level="off"/>
         <Logger name="org.springframework" level="warn"/>
-        <Logger name="intrafilter-logging" level="info"/>
+        <Logger name="intrafilter-logging" level="trace">
+            <ContextMapFilter onMatch="ACCEPT" onMismatch="DENY">
+                <KeyValuePair key="x-trace-request" value="true"/>
+            </ContextMapFilter>
+        </Logger>
+        <Logger name="filter-timing" level="trace">
+            <ContextMapFilter onMatch="ACCEPT" onMismatch="DENY">
+                <KeyValuePair key="x-trace-request" value="true"/>
+            </ContextMapFilter>
+        </Logger>
```

Comment:
Ah. This probably needs to change. The problem with this configuration is that if the `x-trace-request` header isn't present, this logger still won't log. That is, we effectively ignore the log level implicit filter.

Using the `DynamicThresholdFilter` we could get around that and see the behavior we want, but unfortunately, it would apply to all loggers (i.e., all loggers would effectively be set to trace if the `x-trace-request` mapping was present).